### PR TITLE
Gist Import

### PR DESCRIPTION
**Pull request by zebulonj**
Implements functional wireframes for gist (Github) import, with client-side routing.  Depends on update to H-API (see separate pull request on that repository).

Next steps:  add client-side templating {{ mustache }} and use in gist edit view.

### CODE REVIEW

File: content/themes/wpgists/functions.php [hunk 1]
```diff
@@ -106,6 +110,21 @@
 });
 
 /**
+ * Register and enqueue client-side JavaScript.
+ */
+
+function wpgists_scripts() {
+	wp_register_script( 'requirejs', '//cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js' );
```
Comment (danielbachhuber) [inline content/themes/wpgists/functions.php]: I'd prefer to internalize this if possible

**Comment by danielbachhuber** [comment]
Looking good :)

File: content/themes/wpgists/functions.php (hunk 1 above)
Comment (zebulonj) [inline content/themes/wpgists/functions.php]: No problem.  I'll pull requirejs in.  The others (backbone and underscore) can be internalized as well (already in WordPress).

**Comment by zebulonj** [comment]
Client-side libraries are internalized.

**Comment by danielbachhuber** [comment]
@zebulonj Ready to merge then?

**Comment by zebulonj** [comment]
@danielbachhuber Yes.  I think we're at a good point for merge, unless you want to wait for me to get the edit-gist page in first.  I've got that going on a separate branch (local) in which I've integrated Mustache.php (server-side) and Handlebars.js (client-side).

**Comment by zebulonj** [comment]
@danielbachhuber FWIW, I'd lean toward merging and breaking off the edit page as a separate enhancement.

**Pull request merged**